source/NVDAObjects/window/excel.py

```diff
@@ -505,27 +508,51 @@ def navigationHelper(self,direction):
 		cellPosition.Activate()
 		eventHandler.executeEvent('gainFocus',obj)
 
+	@script(
+		gesture="kb:leftArrow"

```

Comment:
In case of a single gesture you do not need to place them and parenthesis on a separate lines, as long as linter does not complain about line length.

Reply:
I changed this as you suggested, however, I think for blind developers it is better to have the paranthesis on a separate line because it is simpler to know when the def function ends. But this might be only my impression as an unexperienced user.